shap/explainers/_tree.py

```diff
@@ -182,7 +185,7 @@ def __init__(
             self.data = data
         if self.data is None:
             feature_perturbation = "tree_path_dependent"
-            #warnings.warn("Setting feature_perturbation = \"tree_path_dependent\" because no background data was given.")
+            logger.warning("Setting feature_perturbation = \"tree_path_dependent\" because no background data was given.")
```

Comment:
Coming back to this PR from a few months ago, I think let's go with `info` rather than `warning` to avoid generating warnings in typical uses of the package.

```suggestion
            logger.info('Setting feature_perturbation = "tree_path_dependent" because no background data was given.')
```